src/main/java/com/johnstarich/moviematcher/models/Rating.java

```diff
@@ -1,15 +1,18 @@
 package com.johnstarich.moviematcher.models;
 
+import com.johnstarich.moviematcher.store.MovieMatcherDatabase;
 import org.bson.types.ObjectId;
 
+import java.util.List;
+
 /**
  * Created by Josue on 4/17/2016.
  */
 public class Rating extends AbstractModel<Rating> {
     public final ObjectId user_id;
     public final ObjectId movie_id;
     public final String comment;
-    public final byte numeric_rating; //ratings are [0,10]
+    public final int numeric_rating; //ratings are [0,10]
```

Comment:
Yes. I was declaring Rating objects with a byte typecast for the numeric_rating, so that was not the problem. 

The problem is that we get "java.lang.RuntimeException: java.lang.RuntimeException: could not set field numeric_rating: Field has type byte got type class java.lang.Integer" as an exception. We get this error when MovieMatcherDatabase.morphium.findByField(...) gets executed.